modules/doc/content/python/testers/XMLDiff.md

```diff
@@ -0,0 +1,23 @@
+# XMLDiff
+
+`XMLDiff` tests compare `XML` output(s) for the test to a reference in the specified
+`gold_dir` folder.
+
+## Options
+
+Test configuration options are added to the `tests` file.
+
+- `xmldiff`: A list of XML files to compare
+- `gold_dir`: The directory where the \"golden standard\" files reside relative to the TEST_DIR: (default: ./gold/)
```

Comment:
```suggestion
- `gold_dir`: The directory where the \"gold standard\" files reside relative to the TEST_DIR: (default: ./gold/)
```

see https://pubmed.ncbi.nlm.nih.gov/16402524/

Reply:
I copy pasted this from somewhere. i ll fix the source